src/test/scala/mesosphere/marathon/api/v2/normalization/AppNormalizationTest.scala

```diff
@@ -631,24 +634,39 @@ class AppNormalizationTest extends UnitTest {
         )
         raw.normalize(configuredNormalizer) should be(raw.copy(role = Some("someCustomRole")))
       }
-
     }
 
     "normalize accepted resource roles" when {
-      val raw = App(
+      val `without-*-role` = App(
+        id = "/foo",
+        cmd = Option("sleep"),
+        acceptedResourceRoles = Some(Set("other"))
+      )
+
+      val `with-*-role` = App(
```

Comment:
Do we also test the following case
```scala
      val `with-*-role` = App(
        id = "/other/foo",
        cmd = Option("sleep"),
        acceptedResourceRoles = Some(Set("*", "other"))
      )
```

I'm still a little confused how `*` is used.

Reply:
ah, so no sanitization needed? Yeah, that's not covered currently, we can add it

Reply:
Ping @timcharper